app/src/main/java/com/duckduckgo/app/email/db/EmailDataStore.kt

```diff
@@ -30,10 +30,6 @@ interface EmailDataStore {
     var emailToken: String?
     var nextAlias: String?
     var emailUsername: String?
-    var inviteCode: String?
-    var waitlistTimestamp: Int
-    var waitlistToken: String?
-    var sendNotification: Boolean
```

Comment:
Is the whole EmailDataStore / EmailEncryptedSharedPreferences still needed?

Reply:
Yes, we still need the rest of the stuff once a user has signed in to be able to generate duck addresses, etc.

Reply:
Makes sense! What about the `cohort` in EmailDataStore?